src/components/NewTripButton.tsx

```diff
@@ -0,0 +1,160 @@
+import { Transition } from "@headlessui/react";
```

Comment:
- create a new component called Home and move it there `src/components/Home/NewTripButton.tsx`
- add export in `src/components/index.ts`